lms/djangoapps/discussion_api/tests/utils.py

```diff
@@ -18,7 +18,12 @@ def callback(request, _uri, headers):
         data along with the data from response_overrides and dummy values for any
         additional required fields.
         """
+        is_new_thread = thread_data.pop("new_thread")
         response_data = make_minimal_cs_thread(thread_data)
+        # pop resp_total as it is not part of response from CS on thread creation
+        # request
+        if is_new_thread:
+            response_data.pop("resp_total")
```

Comment:
Instead of popping the entry after it's created, pass the `is_new_thread` parameter into `make_minimal_s_thread` and change that method so that `resp_total` gets added to the dict only if `is_new_thread` is `False`.